bookwyrm/templates/book/edit_book.html

```diff
@@ -4,6 +4,45 @@
 
 {% block title %}{% if book %}{% blocktrans with book_title=book.title %}Edit "{{ book_title }}"{% endblocktrans %}{% else %}{% trans "Add Book" %}{% endif %}{% endblock %}
 
+{% block scripts %}
+<script defer>
```

Comment:
Depending on where this PR is gonna go, I can help you with JS if you’re seeking assistance; just ping me if you wish.

Just two things for the sake of sharing:

- `defer` will not do anything on inline scripts. There is [more details on <abbr title="Mozilla Developer Network">MDN</abbr> Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer)
- JS could go to an external file. :)

Reply:
i appreciate that! i'll definitely ping if something comes up. i used an inline script just for development, but i didn't know that about defer, thank you!